addons/web_editor/static/src/js/editor/odoo-editor/src/commands/commands.js

```diff
@@ -542,6 +542,9 @@ function deleteTable(editor, table) {
 // absence of a strong test suite, so the whitelist stays for now.
 export const editorCommands = {
     // Insertion
+    insert: (editor, data, mode) => {
```

Comment:
What is the use of this function ? The only place I found in this commit you call it with 'html' as parameter so you could just as well have called `insertHTML` instead.

Reply:
Yes, but then it will trigger you know what.

Reply:
I think the solution to you know what might be to avoid calling the `insertHTML` command when it is not warranted. In particular, I think we might be misusing the `insertHTML` command in the particular case where the `insert` command is called in this commit. Do we really need to call on the full `execCommand` mechanism and such ? Can't we just insert the node manually ? It's only for a `<t t-out="...">`.

Reply:
How could we insert a HTML tag manually (aka without the `execCommand`) but at the same time avoid triggering 'you know what' ??
also keep in mind all the `execCommand` stuff take care of the history and such.

Reply:
This is the `commands.js` file. Wouldn't it work to call `insert` manually in this particular case rather than through `execCommand` ?

Reply:
For now `insert` is not exported outside of the command.js file. I though it was on purpose to keep it protected. that's why I opted for the proxy.

Reply:
will use insertFragment as discussed